server.js

```diff
@@ -140,3 +149,23 @@ var simulatedOccupancyChanges = interval(5000).pipe(
   share()
 );
 agent.subscribe(simulatedOccupancyChanges);
+
+// TODO If this is the first time the DB has heard of the room, populate it
+for (let { num, occupancy } of createRoomViews(initialState)) {
+  Room.findOrCreate({ num }, { occupancy });
+}
+
+// TODO Upon startup, load the store up with room and occupancy data
+Room.find({}).then(rooms => {
+  agent.process({
+    type: "loadRooms",
+    payload: rooms
+  });
+
+  rooms.map(({ num, occupancy }) => {
+    agent.process({
+      type: "setOccupancy",
+      payload: { num, occupancy }
+    });
+  });
+});
```

Comment:
The database contains everything we need to populate the rooms and occupancy parts of our store, but we only change the store through actions, so we `process` these actions, which filter through the store, and our store will reflect the contents of the database.

Bonus: Now if you ever change the database directly, and need the app to pick up those changes, just restart it! Or use a component like that out of MeteorJS to tail the mongo oplog live (disclaimer: just kidding!).